packages/test/test-pairwise-generator/README.md

```diff
@@ -1,3 +1,9 @@
-# @fluid-internal/test-pairwise-generator
+# @fluidframework/test-pairwise-generator
 
 See [examples.spec.ts](./src/test/examples.spec.ts) for usage examples
+
+## Trademark
```

Comment:
How did you know to add this? Which packages should include it?

Reply:
policy check did it